dashboard/app/models/section.rb

```diff
@@ -18,8 +18,7 @@
 #
 # Indexes
 #
-#  index_sections_on_code     (code) UNIQUE
-#  index_sections_on_user_id  (user_id)
```

Comment:
This is a problem, we still need to be able to find a user's (teacher's) sections efficiently, so the index should be kept.